source/crud.txt

```diff
@@ -0,0 +1,118 @@
+=====================
+CRUD Operations Guide
+=====================
+
+.. default-domain:: mongodb
+
+.. toctree::
+   :caption: CRUD Operations
+   :hidden:
+
+   /crud/read-operations
+   /crud/write-operations
+   /crud/query-document
+   /crud/returned-data
+   /crud/compound-operations
+
+Overview
+--------
+
+CRUD (Create, Read, Update, Delete) operations allow you to work with
+the data stored inside of MongoDB. CRUD operations fall into two
+categories: read operations and write operations. All CRUD operations
+offer customizable options to tailor behavior for your application's
+specific needs. CRUD operations are all asynchronous.
```

Comment:
These last two sentences seem a little bit tacked on. I think we could expand each into a separate paragraph.

- **Options:** the sentence you have is good. Maybe we could use the space to point to some of the read/write operation sub pages that show different options? (e.g. **Sort Results**)
- **Async:** what does asynchronous mean, why does it matter, and how does it affect driver code? Async code is notorious tough for beginners so we should take care to introduce it well.

Reply:
Good point. I've expanded both of these points into subsections of their own right. Right now I feel like the customization section does a decent job of explaining some basic options/pointing you towards node driver doc pages, but the async section is a little tougher. Would appreciate suggestions on:
- section name
- tweaks to how I approach the explanation, which is a bit... hackneyed right now